.github/workflows/snyk.yml

```
@@ -2,47 +2,75 @@ name: Snyk Security Scan
-
+    defaults:
+      run:
+        working-directory: ./api-gateway
     steps:
-      - name: Checkout Code
+      - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Setup Node.js
+      - name: Set up Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)